sass/components/rtl_support/_badges.scss

```diff
@@ -0,0 +1,17 @@
+// RTL Badges
+.rtl {
+	span.badge {
+		margin-left: initial;
+		margin-right: 14px;
+		float: left;
+
+		&.new:after {
+			content: " جدید";
```

Comment:
@arminmon You're probably right, but it's just one automatic call when the page loads and another method triggered manually by developer, so it wouldn't take so much memory.